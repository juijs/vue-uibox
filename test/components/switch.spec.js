import { mount } from 'vue-test-utils'
import UixSwitch from '@/components/switch'

describe('switch.js', () => {
    const wrapper = mount(UixSwitch, {
        propsData: {
            size: 'small',
            checked: false
        }
    });

    it('snapshot!!', () => {
        wrapper.vm.$nextTick(() => {
            expect(wrapper.vm.$el).toMatchSnapshot();
        });
    });
})