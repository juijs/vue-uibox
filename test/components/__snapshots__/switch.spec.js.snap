// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`switch.js snapshot!! 1`] = `
<div
  class="switch small inner"
>
  <div
    class="area"
  >
    <div
      class="bar"
    >
      <div
        class="left"
      />
      <div
        class="right"
      />
    </div>
  </div>
  <div
    class="handle"
  />
</div>
`;
